App/Containers/ThreadsList.js

```diff
@@ -17,6 +18,13 @@ import PreferencesActions from '../Redux/PreferencesRedux'
 import TextileNodeActions from '../Redux/TextileNodeRedux'
 import { getThreads } from '../Redux/PhotoViewingSelectors'
 
+function isEmpty(obj) {
```

Comment:
This is used to check if we have an empty object (the default). Couldn't find this one already built in... apparently this is safe than `Object.keys(obj).length`?